brainscore_language/model_helpers/huggingface.py

```diff
@@ -209,7 +209,7 @@ def predict_next_word(self, base_output: CausalLMOutput):
         logits = base_output.logits
         pred_id = torch.argmax(logits, axis=2).squeeze()
         # Note that this is currently only predicting the next *token* which might not always be entire words.
-        last_model_token_inference = pred_id[-1].tolist()
+        last_model_token_inference = pred_id[-1].tolist() if len(pred_id.size()) > 0 else pred_id.item()
```

Comment:
could this not just be `pred_id.item()` in all cases?

Reply:
No, it fails when handed a >0-dim tensor, which occurs in other benchmarks.